common/djangoapps/third_party_auth/management/commands/tests/test_saml.py

```diff
@@ -348,29 +347,21 @@ def test_run_checks_outdated_configs(self, mock_set_custom_attribute):
 
         self.assertIn('[WARNING]', output)
         self.assertIn('test-provider', output)
-        self.assertIn(
-            f'id={old_config.id} which should be updated to the current SAML config (id={new_config.id})',
-            output
+        outdated_msg = (
+            f'has outdated SAML config (id={old_config.id}) which should be updated to '
+            f'the current SAML config (id={new_config.id})'
         )
+        self.assertIn(outdated_msg, output)
         self.assertIn('CHECK SUMMARY:', output)
         self.assertIn('Providers checked: 2', output)
+        self.assertIn('Informational only:', output)
+        self.assertIn('Slug mismatches: 1', output)
+        self.assertIn('Missing configs: 1', output)  # 1 null from setUp
+        self.assertIn('Issues requiring attention:', output)
         self.assertIn('Outdated: 1', output)
+        self.assertIn('Total issues requiring attention: 1', output)  # 1 outdated only
```

Comment:
I find most of these assertions distracting from what we are actually testing. Can we strip this down to the following?:
```suggestion
        self.assertIn('Outdated: 1', output)
        self.assertIn('Total issues requiring attention: 1', output)
```

Please do something similar to the other tests, so it is more clear what each test is for.